pre-lollipop-activity-transition/src/main/java/com/kogitune/activity_transition/core/TransitionAnimation.java

```diff
@@ -73,11 +76,23 @@ private static void runEnterAnimation(MoveData moveData, TimeInterpolator interp
     private static void setImageToView(View toView, String imageFilePath) {
         Bitmap bitmap;
         if (bitmapCache == null || (bitmap = bitmapCache.get()) == null) {
+            synchronized (lock) {
+                while (isReady) {
```

Comment:
This code seems quite hard to read. I will add a commit to explain this code well.

For quick response, it must have `while(isReady)` inside of synchronize block. I should've put [this line](https://github.com/takahirom/PreLollipopTransition/pull/18/files#diff-ab884a8ff1acc50ce2b9db3d7c0d8bccR43) inside of the thread and enveloped synchronize block. That's why this code seems awkward. This code wants to lock the flag, not its lock object.